scene/3d/mesh_instance_3d.cpp

```diff
@@ -380,6 +380,13 @@ void MeshInstance3D::_mesh_changed() {
 		}
 	}
 
+	int surface_count = mesh->get_surface_count();
+	for (int surface_index(0); surface_index < surface_count; ++surface_index) {
```

Comment:
Just a style nitpick, we don't use the ``surface_index(0)``-style initialization in for loops we do the old fashioned ``int surface_index = 0`` for consistency, we should do the same here.

Reply:
I thought Juan said `master` can use C++14/17 now.

Reply:
I need to do some tests with these changes. It seems like the solution doesn't work anymore.